schema-engine/sql-introspection-tests/tests/views/postgresql.rs

```diff
@@ -783,17 +763,14 @@ async fn re_intro_keeps_the_map(api: &mut TestApi) -> TestResult {
     api.expect_re_introspected_datamodel(input, expected).await;
 
     let expected = expect![[r#"
-        [
-          {
-            "code": 23,
-            "message": "These views were enriched with `@@map` information taken from the previous Prisma schema.",
-            "affected": [
-              {
-                "view": "B"
-              }
-            ]
-          }
-        ]"#]];
+        *** WARNING ***
+
+        These views were enriched with `@@map` information taken from the previous Prisma schema:
+          - B
+
+        These views were enriched with `@@map` information taken from the previous Prisma schema:
+          - B
+    "#]];
```

Comment:
This is duplicate warning is unexpected to me

Reply:
Yeah, fixed